src/openzaak/components/catalogi/admin/zaaktypen.py

```diff
@@ -307,6 +323,15 @@ def get_related_objects(self, obj):
 
         return resource_list, id_list
 
+    def response_post_save_change(self, request, obj):
```

Comment:
I think this method is not needed anymore

Reply:
Removed